source/platforms/bitbucket_server/_tests/_bitbucket_server_git.test.ts

```diff
@@ -135,4 +136,12 @@ describe("the dangerfile gitDSL - BitBucket Server", async () => {
     const promise = bbs.findTypeOfLine(gitDSL, 0, "jest.eslint.config.js")
     await expect(promise).rejects
   })
+
+  it("writes a JSON DSL fixture", async () => {
+    const fakeSource = new FakeCI({})
+    const dataSent = await jsonDSLGenerator(bbs, fakeSource, {} as any)
+    dataSent.settings.github.accessToken = "12345"
+
+    writeFileSync(pathJoin(fixtures, "bbs-dsl-input.json"), JSON.stringify(dataSent, null, "  "), "utf8")
```

Comment:
A bit late to the game now, but does this file have any usage yet? This test produces local (looks like formatting only) changes for me after a test run and this test doesn't seem to have any verifications? Can we remove it?

Reply:
I've fixed that in #778 

But yeah, it does - it's used in Danger Swift and Kotlin, it's means to act as a reference for other language runners